vendor/assets/javascripts/initializers.js

```diff
@@ -40,7 +40,14 @@ $(document).on('turbolinks:load', function() {
   });
   $('.js-dropzone').each(function() {
     $(this).dropzone({
-      dictDefaultMessage: 'Drop files or click here to upload.'
-    })
+      dictDefaultMessage: 'Drop files or click here to upload.',
+      init: function() {
+        this.on("addedfile", function(files) {
+          $('.js-dropzone-done').addClass('disabled');
```

Comment:
Is it possible to not make this a global find, but scope it to the dropzone instance? See if the callback is passed any additional parameters. I bet it at least gets an `event` param and you could inspect that and probably do something like `$(event.target).find('.js-dropzone-done')`